specs/IR-0001/suite/test_spec_ir0001.py

```
@@ -0,0 +1,381 @@
+
+    def test_no_escape_hatch(self):
+        """BEH/INV/IFACE 条款与 AC then 禁逃生从句。"""
+        s = read(SPEC)
+        lines = [l for l in s.splitlines()
+                 if re.match(r"^\s*(- )?(id: AC-|then:|(?:INV|BEH|IFACE)-\d+:)", l.strip())]
+        for l in lines:
+            hit = [w for w in self.ESCAPE_HATCH if w in l]
+            self.assertEqual(hit, [], f"逃生舱从句 {hit}: {l[:60]}——强义务被掏空")
+
+    # ---- S6（前置条件堆叠）补强：触发条件层不得偷改验收范围 ----
+    BEH_COND_STACK = ["且", "预检", "就绪", "齐备", "配额"]
```

Ruff (0.16.2)

[warning] Mutable default value for class attribute

(RUF012)